src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -300,4 +347,8 @@
                                      (log/error "failed to send transaction"
                                                 {:event  :wallet/send-transaction
                                                  :error  error
-                                                 :params request-params}))}]})))
+                                                 :params request-params})
+                                     (rf/dispatch [:toasts/upsert
```

Comment:
I wonder is this error human readable enough? 🤔

Reply:
Probably not, I think it is good enough for the time being because for bridging different kind of errors may appear even if routes are found and everything appears to be fine but other errors are encountered after calling the endpoint (not enough gas, not reaching minimum amount of tokens to be bridged) and is good for knowing what's happening. Maybe we can show toeast this only for briding transactions errors.

Reply:
For sure it's good to have for now, was just curious 👌

Reply:
Yes, I thought this wasn't needed until I saw some new errors when testing bridging that are "discovered" when trying to execute the transaction, would be difficult for QA (or users) to know why bridging fails in some situations without showing an error. I'm sure we will come back to this in the future, we should probably align with desktop team.